docs/source/upgrade-guide/index.md

```diff
@@ -28,7 +28,21 @@ It is usually better and quicker to move your items into new locations and copy
 ```
 ````
 
-(volto-upgrade-guide-18.x.x)=
+(volto-upgrade-guide-19.x.x)=
+
+## Upgrading to Volto 19.x.x
+
+(volto-upgrade-guide-19.x.x)=
+
+### Removed support for loading config from project
+
+This breaking change fulfills this [PLIP](https://github.com/plone/volto/issues/6396).
```

Comment:
Off-topic, but something to consider: since teams are driving PLIPs these days, with no more FWT to review them, should we number our PLIPs to designate the affected component or package? For example, `PLIP[volto] 6396`. Eventually we'll get number collisions across all parts of Plone. Can we add this as a topic at an appropriate sprint, or create a PLIP for PLIPs?

```suggestion
This breaking change fulfills [PLIP 6396](https://github.com/plone/volto/issues/6396).
```